### ISSUE TITLE
[Bug]: Live ISO will not load

### ISSUE THREAD

**Issue by kleytr6s** [ghostbsd/ghostbsd-src#172]
### Describe the bug

I have tried the latest stable and nightly releases all of them stop loading at the line shown in the picture attached. Also, when the loading stops lines quickly appear across the screen and disappear.
![ghostbsd bug](https://user-images.githubusercontent.com/16599146/153802463-923bf824-61bf-4a4d-b4d0-d983df08c7c8.jpg)

### How to reproduce the bug?

N/A

### Expected behavior

N/A

**Comment by ericbsd** [comment]
Without any system information that can't be fixed.

**Comment by kleytr6s** [comment]
What system info do you need?

**Comment by ericbsd** [comment]
Computer hardware information.

**Comment by kleytr6s** [comment]
Asus X553SA
Intel Celeron N3050 64bit
4GB Ram
15.6 display 1366 x 768
DVD Drive
500GB HDD
HDMI
VGA
USB 2.0, 3.0
Ethernet
WIFI

**Comment by github12101** [comment]
Cannot reproduce. Tested on Toshiba laptop, both stable and development ISOs are working.
Perhaps this is your RAM?
Please check with memtest86/memtest86+.

**Comment by kleytr6s** [comment]
I do not think it is the RAM. Every live BSD distro I try will not load. Live Linux distros work. And, I run Windows daily without RAM/Blue screen errors, etc.

**Comment by github12101** [comment]
Hmm, I might be wrong, but in that case I suggest that you file a bug at the source, FreeBSD. GhostBSD just takes things upstream from FreeBSD.
File a bug against FreeBSD not booting on your machine. Provide them with all information required, and of course test if indeed latest FreeBSD is not booting. Try older images too, or development ones, if available. Good luck.
